tests/ui/return/return-impl-trait-bad.stderr

```diff
@@ -53,6 +53,7 @@ LL |     "don't suggest this, because the generic param is used in the bound."
    |
    = note: expected type parameter `T`
                    found reference `&'static str`
+   = note: the type of `T` is chosen by the caller and may be a type that is different than `T`
```

Comment:
> may be a type that is different than `T`

Hmm... typo?

Reply:
Fixed it

Reply:
Sorry, I was not clear with what I meant here. 

> the type of `T` is chosen by the caller and may be a type that is different than `T`

This sentence mentions `T` twice, which does not make sense.